### DISCUSSION TITLE
User enabled after permanent disabling

### DISCUSSION THREAD

**Discussion by SShahmuradli** [keycloak/keycloak#17604] [Q&A] [answered]
Hi, i am using permanent lockout after 5 times wrong password entering. We can see user_temporarily_disabled in events, but in some time(about 1-2 minutes) i see that user can login and he is not blocked. What is the problem? Thanks in advance

![image](https://user-images.githubusercontent.com/113012735/224572385-368f1192-e3d5-4dca-bfc6-c2f2a8655995.png)
https://stackoverflow.com/questions/75715668/user-enabled-after-permanent-disabling-in-keycloak

**Comment by SShahmuradli** [comment, marked as answer]
Decreasing Quick Login Check Milli Seconds  solved the problem. User was blocked by quick login, not by 5 time wrong password

**Comment by atul-agg** [comment]
I noticed that user is getting "Temporarily locked" along with being disabled with Permanent lockout. Is this correct behavior?
